errors/errors.js

```diff
@@ -0,0 +1,20 @@
+exports.badPathErrorHandler = (req, res, next) => {
+    res.status(404).send({ msg: "Path not found" });
+};
+
+exports.postgresErrorHandler = (err, req, res, next) => {
+    if (err.code) {
+        res.status(400).send({msg: "Bad request"})
+    } else {
+        next(err);
+    }
+};
+
+exports.customErrorHandler = (err, req, res, next) => {
+    if (err.status && err.msg) {
+        res.status(err.status).send({msg: err.msg})
+    } else {
+        next(err);
+    }
+};
+
```

Comment:
Good job extracting these! 

You can add a 500 error handler too - you don't have to test for this but it should be your last 'resort' error handler. If all else fails.